material_admin.theme

```diff
@@ -162,3 +167,14 @@ function material_admin_preprocess_links__dropbutton(&$variables) {
 function material_admin_preprocess_color_scheme_form(&$variables) {
   $variables['#attached']['library'][] =  'material_admin/color';
 }
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function material_admin_theme_suggestions_page_alter(array &$suggestions, array $variables) {
+  $route_match = \Drupal::routeMatch()->getRouteName();
+  $portal_paths = array('user.pass', 'user.login', 'user.register');
```

Comment:
@mortenson this is the method I came up with here, it's pretty manual.. but I couldn't think of a better way without it showing this template for all /user/* which I don't want to do.